src/components/workflowEditor/workflowEditor.css

```diff
@@ -80,3 +83,24 @@
 .action-card:hover {
     background: var(--N50);
 }
+
+.webhook-tippy-card-container .webhook-tippy-card {
+    background-color: #2d3452;
```

Comment:
Can we use the color var for this or introduce one if not available?

Reply:
done